ReGoap/Planner/PlanDebugger1.cs

```diff
@@ -0,0 +1,378 @@
+﻿// ------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//     Runtime Version: 14.0.0.0
+//  
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+// ------------------------------------------------------------------------------
+namespace Assets.ReGoap.Planner
+{
+    using System.Linq;
+    using System.Text;
+    using System.Collections.Generic;
+    using System;
+    
+    /// <summary>
+    /// Class to produce the template output
+    /// </summary>
+    
+    #line 1 "E:\works\programs\GameProjects\TestGround\TestReGOAP\Assets\ReGoap\Planner\PlanDebugger.tt"
```

Comment:
What's this file? why is there a full path of your hard drive here?
Also why is it named PlanDebugger**1**?

Reply:
PlanDebugger.tt is a [T4](https://msdn.microsoft.com/en-us/library/ee844259.aspx) code generation file, which is used by VisualStudio to generate this PlanDebugger1.cs.

PlanDebugger1.cs is used to create the logs for graphviz;